.github/workflows/publish.yml

```
@@ -0,0 +1,43 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
```

Maybe something shorter and does not include emoji (shocking coming from me, I know)? This name is the same name used in badge, in case we decide to include this in a badge in the future.
```suggestion
name: Release
```